doc/en/user/source/community/wps-longitudinal-profile/index.rst

```diff
@@ -0,0 +1,77 @@
+.. _wpslongitudinal:
+
+WPS longitudinal profile process
+================================
+
+WPS longitudinal profile process provides the ability to calculate an altitude profile for the specified linestring.
+
+**In addition, the process can:**
+
+* Reproject result to different CRS
+* Adjust altitude profile based on additional layer
+
+Installing the WPS longitudinal profile process
+-----------------------------------------------
+
+ #. If you haven't done already, install the WPS extension: :ref:`wps_install`.
+
+ #. Download the WPS longitudinal profile process extension from the `nightly GeoServer community module builds <https://build.geoserver.org/geoserver/main/community-latest/>`_.
+
+    .. warning:: Make sure to match the version of the extension to the version of the GeoServer instance!
+
+ #. Extract the contents of the archive into the ``WEB-INF/lib`` directory of the GeoServer installation.
+
+
+Module description
+------------------
+
+This module provides longitudinal profile process.
+The process splits provided geometry (for example linestring) into segments of no more then provided distance length.
+Then evaluates altitude for each point and builds longitudinal profile.
+If adjustment layer name is provided, altitude will be adjusted by searching feature that contains corresponding point,
+and getting it's altitude attribute, further subtracting it from altitude received from coverage.
+If projection parameter is provided, points of profile will be reprojected to target CRS, otherwise to CRS
```

Comment:
isn't this right?

```suggestion
If targetProjection parameter is provided, points of profile will be reprojected to target CRS, otherwise to CRS
```

Reply:
fixed